storagetransfer/requirements-test.txt

```
@@ -5,5 +5,5 @@ boto3==1.34.134
 google-cloud-pubsub==2.21.5
 google-cloud-storage==2.9.0; python_version < '3.7'
 google-cloud-storage==2.9.0; python_version > '3.6'
-google-cloud-secret-manager==2.16.1
+google-cloud-secret-manager==2.21.1
```

Remove bump on sample with larger issues outside this change

```suggestion
google-cloud-secret-manager==2.16.1
```